crates/astria-conductor/src/conductor.rs

```diff
@@ -152,13 +155,31 @@ impl Conductor {
             let (shutdown_tx, shutdown_rx) = oneshot::channel();
             shutdown_channels.insert(Self::DATA_AVAILABILITY, shutdown_tx);
             let block_verifier = BlockVerifier::new(sequencer_client_pool.clone());
+            let sequencer_namespace = {
+                use sequencer_client::SequencerClientExt as _;
+
+                let client = sequencer_client_pool.get().await?;
+                let chain_id = client
+                    .latest_sequencer_block()
```

Comment:
This should have a retry IMO.

Note that this comment is obsolete if you follow my comment re sequencer being absent in firm-only setups.